PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -230,7 +230,12 @@ def miniAOD_customizeCommon(process):
                   'RecoEgamma.PhotonIdentification.Identification.cutBasedPhotonID_Spring15_50ns_V1_cff',
                   'RecoEgamma.PhotonIdentification.Identification.mvaPhotonID_Spring15_25ns_nonTrig_V2p1_cff',
                   'RecoEgamma.PhotonIdentification.Identification.mvaPhotonID_Spring15_50ns_nonTrig_V2p1_cff']
```

Comment:
shouldn't the mvaPhotonID_Spring16_nonTrig_V1_cff  and cutBasedPhotonID_Spring16_V2p2_cff be added here to save them in miniAOD?